doc/examples/uploadsafe.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+import pathlib
+import shutil
+import pymonetdb
+
+class MyUploader(pymonetdb.Uploader):
+    def __init__(self, dir):
+        self.dir = pathlib.Path(dir)
+
+    def handle_upload(self, upload, filename, text_mode, skip_amount):
+        # security check
+        path = self.dir.joinpath(filename).resolve()
+        if not str(path).startswith(str(self.dir.resolve())):
+            return upload.send_error('Forbidden')
+        # open
+        tw = upload.text_writer()
+        f = open(path)
```

Comment:
probably better to use `with open(path) as f`

Reply:
Thanks, done.